scripts/travis-ci.sh

```diff
@@ -0,0 +1,7 @@
+docker build --tag=digitalbitbox/bitbox-base .
+docker run -v ${TRAVIS_BUILD_DIR}:/opt/go/src/github.com/digitalbitbox/bitbox-base/ \
+        -i digitalbitbox/bitbox-base \
+        bash -c "make -C \$GOPATH/src/github.com/digitalbitbox/bitbox-base/middleware ci" \
+        bash -c "make -C \$GOPATH/src/github.com/digitalbitbox/bitbox-base/middleware native" \
+        bash -c "make -C \$GOPATH/src/github.com/digitalbitbox/bitbox-base/bbbfancontrol"
```

Comment:
This is actually missing `tools/`. I.e it should be:
```
          bash -c "make -C \$GOPATH/src/github.com/digitalbitbox/bitbox-base/tools/bbbfancontrol"
```

(note extra `tools/`)

Reply:
Fixed.